pkg/KubeArmorAuditPolicy/PROJECT

```diff
@@ -0,0 +1,7 @@
+domain: accuknox.com
```

Comment:
use kubearmor.com instead of accuknox.com
please use this in the other files too.